### PR TITLE
GC17 Icons

### PR DESCRIPTION
**Pull request by jack-han-115**
Fixed issue #<Mantis issue number>:
New feature #<Mantis issue number>:
Dev:

### CODE REVIEW

File: application/extensions/TopbarWidget/views/includes/questionTopbarLeft_view.php [hunk 1]
```diff
@@ -35,4 +35,33 @@
 $this->render('includes/previewQuestionButton_view', get_defined_vars());
 ?>
 
+<<<<<<< HEAD
```
Comment (twilligls) [inline application/extensions/TopbarWidget/views/includes/questionTopbarLeft_view.php:38]: merge conflict

**Comment by Shnoulle** [comment]
Since i see this : we remove fontawesome from default theme.

What is the new one ?

Else : about all of this update : since FA can be used too on QuestionTheme : maybe it's a good idea to start a wiki page of API update of Theme ?

**Comment by olleharstedt** [comment]
> Else : about all of this update : since FA can be used too on QuestionTheme : maybe it's a good idea to start a wiki page of API update of Theme ?

Might be good, yes. Do we have one for LS3 -> LS5 already?

**Comment by olleharstedt** [comment]
This is our new icon set: https://remixicon.com/

**Comment by olleharstedt** [comment]
:cry:

**Pull request merged**